components/brave_wallet/browser/fil_transaction.h

```diff
@@ -37,9 +37,11 @@ class FilTransaction {
   int64_t gas_limit() const { return gas_limit_; }
   std::string max_fee() const { return max_fee_; }
   FilAddress to() const { return to_; }
+  FilAddress from() const { return from_; }
   std::string value() const { return value_; }
 
   void set_to(FilAddress to) { to_ = to; }
+  void set_from(FilAddress from) { from_ = from; }
```

Comment:
setters' arguments should be const T& or std::move into fields

Reply:
done